docs/4.1/quickstart__archive.md

```diff
@@ -0,0 +1,279 @@
+# Quick Start Guide
+
+Welcome to the Teleport Quick Start Guide!
+
+The goal of this document is help you start using Teleport.
+There are three types of services Teleport nodes can run: `nodes`, `proxies`
+and `auth servers`.
+
+- **Auth servers** store user accounts
+  and provide authentication and authorization services for every node and
+  every user in a cluster.
+- **Proxy servers** route client connection requests to the appropriate node
+  and serve a Web UI which can also be used to log into SSH nodes. Every
+  client-to-node connection in Teleport must be routed through the proxy.
+- **Nodes** are regular SSH nodes, similar to the `sshd` daemon you may be
+  familiar with. When a node receives a connection request, the request is
+  authenticated through the cluster's auth server.
+
+The `teleport` daemon runs all three of these services by default. This Quick Start Guide will
+be using this default behavior to create a cluster and interact with it
+using Teleport's client-side tools:
+
+| Tool    | Description
+|---------|-------------
+| tctl    | Cluster administration tool used to invite nodes to a cluster and manage user accounts.
+| tsh     | Similar in principle to OpenSSH's `ssh`. Used to login into remote SSH nodes, list and search for nodes in a cluster, securely upload/download files, etc.
+| browser | You can use your web browser to login into any Teleport node by opening `https://<proxy-host>:3080`.
+
+## Installing and Starting
+
+Teleport runs on Linux, MacOS and Teleport 3.0 now supports Windows for running `tsh login`.
+You can download pre-built binaries from [here](https://gravitational.com/teleport/download/)
+or you can [build it from source](https://github.com/gravitational/teleport).
+
+After downloading the binary tarball, run:
+
+```bash
+$ tar -xzf teleport-binary-release.tar.gz
+$ cd teleport
+$ sudo ./install
+```
+
+This will copy Teleport binaries to `/usr/local/bin`.
+
+Let's start Teleport on a single-node. First, create a directory for Teleport
+to keep its data. By default it's `/var/lib/teleport`. Then start `teleport` daemon:
+
+```bash
+$ mkdir -p /var/lib/teleport
+$ sudo teleport start
+
+[AUTH]  Auth service is starting on 0.0.0.0:3025
+[PROXY] Reverse tunnel service is starting on 0.0.0.0:3024
+[PROXY] Web proxy service is starting on 0.0.0.0:3080
+[PROXY] SSH proxy service is starting on 0.0.0.0:3023
+[SSH]   Service is starting on 0.0.0.0:3022
+```
+
+At this point you should see Teleport print listening IPs of all 3 services into the console.
+
+Congratulations - you are now running Teleport!
+
+If you wish to deploy Teleport inside a Docker container:
+
+```
+# This command will pull the Teleport container image for version 4.0.0-rc.2
+# Replace 2.7.3 with the version you want to run:
```

Comment:
2.7.3 -> 4.0.0-rc.2 (number in example doesn't match the command)

Also we probably don't want to suggest people deploy RC versions of Teleport. Change all to 4.1.0 or something?